tests/rptest/services/utils.py

```diff
@@ -6,20 +6,46 @@
 from typing import Generator, Optional
 
 from rptest.clients.kubectl import KubectlTool, KubeNodeShell
-from rptest.services.cloud_broker import CloudBroker
+
+
+class Stopwatch():
+    def __init__(self) -> None:
+        self._start = 0
+        self._end = 0
+
+    def start(self):
+        self._start = time.time()
+        self._end = self._start
+
+    def stop(self):
```

Comment:
My Casio would call this operation "Split" :)

Reply:
Agreed. This is not a stop in normal sense.